src/test/java/org/mastodon/mamut/io/importer/labelimage/ui/ImportSpotsFromImgPlusViewDemo.java

```diff
@@ -0,0 +1,32 @@
+package org.mastodon.mamut.io.importer.labelimage.ui;
+
+import net.imagej.ImgPlus;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.type.numeric.real.FloatType;
+import org.mastodon.mamut.ProjectModel;
+import org.mastodon.mamut.io.importer.labelimage.demo.DemoUtils;
+import org.mastodon.mamut.model.Model;
+import org.scijava.Context;
+import org.scijava.command.CommandService;
+import org.scijava.ui.UIService;
+
+public class ImportSpotsFromImgPlusViewDemo
+{
+
+	public static void main( String[] args )
+	{
+		@SuppressWarnings( "all" )
+		Context context = new Context();
+		UIService ui = context.service( UIService.class );
+		CommandService cmd = context.service( CommandService.class );
+
+		Img< FloatType > img = ArrayImgs.floats( 10, 10, 10, 1 );
+		ImgPlus< FloatType > imgPlus = new ImgPlus<>( img, "Dummy Image" );
+		Model model = new Model();
+		ProjectModel projectModel = DemoUtils.wrapAsAppModel( img, model, context );
+
+		ui.showUI();
+		cmd.run( ImportSpotsFromImgPlusView.class, true, "imgPlus", imgPlus, "projectModel", projectModel );
+	}
+}
```

Comment:
An improved version of the test that allows to fully run the command and see the results:
(This demo can also be easily adapted for the ImportSpotsFromBdv... demo.)
```suggestion
package org.mastodon.mamut.io.importer.labelimage.ui;

import java.util.Arrays;
import java.util.List;

import net.imglib2.img.Img;
import net.imglib2.img.ImgView;
import net.imglib2.img.display.imagej.ImageJFunctions;
import net.imglib2.type.numeric.real.FloatType;
import net.imglib2.view.Views;

import org.mastodon.mamut.MainWindow;
import org.mastodon.mamut.ProjectModel;
import org.mastodon.mamut.io.importer.labelimage.demo.DemoUtils;
import org.mastodon.mamut.model.Model;
import org.mastodon.mamut.views.bdv.MamutViewBdv;
import org.scijava.Context;
import org.scijava.command.CommandService;
import org.scijava.ui.UIService;

public class ImportSpotsFromImgPlusViewDemo
{

	public static void main( String[] args )
	{
		@SuppressWarnings( "all" )
		Context context = new Context();
		UIService ui = context.service( UIService.class );
		CommandService cmd = context.service( CommandService.class );

		Img< FloatType > image = generateExampleImage();

		// show image in ImageJ
		ui.showUI();
		ImageJFunctions.wrap( image, "label image" ).show(); 
		// open the image in Mastodon
		Model model = new Model();
		ProjectModel projectModel = DemoUtils.wrapAsAppModel( image, model, context );
		new MainWindow( projectModel ).setVisible( true );
		projectModel.getWindowManager().createView( MamutViewBdv.class );
		// run import spots command
		cmd.run( ImportSpotsFromImgPlusView.class, true, "projectModel", projectModel );
	}

	private static Img< FloatType > generateExampleImage()
	{
		double[] center = { 40, 50, 60 };
		double[][] givenCovariance = {
				{ 400, 20, -10 },
				{ 20, 200, 30 },
				{ -10, 30, 100 }
		};
		long[] dimensions = { 100, 100, 100 };
		int background = 0;
		int pixelValue = 1;
		Img< FloatType > frame = DemoUtils.generateExampleImage( center, givenCovariance, dimensions, background, pixelValue );
		List< Img< FloatType > > twoIdenticalFrames = Arrays.asList( frame, frame );
		return ImgView.wrap( Views.stack( twoIdenticalFrames ) );
	}
}
```

Reply:
Thanks for this demo. This is of course very useful. I changed it a bit so that the axes in the ImageJ view match and that no user interaction in the BDV is necessary to see the ellipsoid.

I added a BDV demo as well.